tabpy-server/setup.py

```diff
@@ -25,21 +26,26 @@
               'tabpy_server.psws',
               'tabpy_server.static'],
     package_data={'tabpy_server.static': ['*.*'],
-                  'tabpy_server': ['startup.*', 'state.ini.template']},
+                  'tabpy_server': ['startup.*', 'state.ini']},
     license='MIT',
     install_requires=[
+        'backports_abc',
+        'cloudpickle',
+        'configparser',
+        'decorator',
         'future',
         'futures',
+        'genson',
+        'jsonschema>=2.3.0',
+        'mock',
+        'numpy',
+        'python-dateutil',
+        'pyOpenSSL',
         'requests',
+        'singledispatch',
         'simplejson',
-        'Tornado-JSON',
-        'tornado',
-        'cloudpickle',
-        'decorator',
-        'python-dateutil',
-        'genson',
-        'jsonschema',
-        'tabpy_client',
-        'numpy'
-    ]
+        'tornado==5.1.1',
```

Comment:
Does this really need to be pinned to this particular version? That's a rather strict requirement

Reply:
Testing this on different OSes and Python versions we found in some cases some other packages may bring an alpha version of Tornado (like 6a1) which causes failure at server starting due to breaking changes in Tornado 6+